### DISCUSSION TITLE
1.22.0

### DISCUSSION THREAD

**Discussion by dani-garcia** [dani-garcia/vaultwarden#1802] [General]
⚠️ **Reminder**: If you are still using the `bitwardenrs/server*` Docker images, you need to migrate to the new `vaultwarden` image. Check https://github.com/dani-garcia/vaultwarden/discussions/1642 for an explanation. The old images are deprecated and will stop being updated after 1.23.0.

- Added `sends_allowed` option to disable Send functionality.
- Added support for hiding the senders email address.
- Added Send options policy.
- Added support for password reprompt.
- Switched to the new attachment download API.
- Send download links use a token system to limit their downloads.
- Updates to the icon fetching.
- Support for webauthn.
- The admin page now shows which variables are overridden.
- Updated dependencies and docker base images.
- Now RSA keys are generated with the included openssl instead of calling to the openssl binary.
- The web vault doesn't require accepting the terms are conditions now, which weren't applicable for a self hosted server.

<hr /><em>This discussion was created from the release <a href='https://github.com/dani-garcia/vaultwarden/releases/tag/1.22.0'>1.22.0</a>.</em>

**Comment by fxzxmic** [comment]
Waiting for webauthn and alpine docker image.

**Reply by dani-garcia** [reply]
I've created a new minor release (1.22.1) to fix the issue with the alpine images. Once docker hub builds this image, it should contain the webauthn changes.

**Reply by fxzxmic** [reply]
Nice!

**Reply by fxzxmic** [reply]
@dani-garcia No Alpine image yet?

**Comment by BlackDex** [comment]
@fxzxmic, it could take a while for those images to be build. Have some patients :)

**Reply by fxzxmic** [reply]
I've been waiting the whole day for the latest version. I can't wait any more. 😊

**Reply by BlackDex** [reply]
Haha, it is live just 14min ago ;)

**Reply by fxzxmic** [reply]
😊

**Comment by spudje** [comment]
Thanks for the new release again. I need some help with the password reprompt. I can edit this setting per entry in the Chrome extension. But I don't see options for this in the webvault? And can I turn this off for all my entries at once?

**Reply by BlackDex** [reply]
Password repromt isn't in this web-vault version. Thats enabled in v2.21.0

**Reply by spudje** [reply]
OK, thanks! I never really grasped how rs/vaultwarden version numbering related to the main bitwarden versioning. So is that an update we can expect soon or not?

**Reply by BlackDex** [reply]
Well, for v2.21.0 we need to fix some items on the server side. I'm already working on some items.
But i bumped into an issue with registration.

Also, the web-vault has it's own versioning.
Bitwarden it self also has different versions for the backend and frontend.
The web-vault (frontend) latest version is now v2.21.0, the API (bitwarden backend) currently is at v1.41.4.

See: https://github.com/bitwarden/server/blob/master/scripts/bitwarden.sh#L37-L38

And also: https://hub.docker.com/r/bitwarden/api/tags?page=1&ordering=last_updated
And: https://hub.docker.com/r/bitwarden/web/tags?page=1&ordering=last_updated

**Reply by coronas2k** [reply]
I downgraded back to 1.21.0, while I appreciate the increased security. It's affecting my workflow. I'll patiently await an update to the backend to disable it. Thanks for all your efforts!

**Reply by BlackDex** [reply]
> I downgraded back to 1.21.0, while I appreciate the increased security. It's affecting my workflow. I'll patiently await an update to the backend to disable it. Thanks for all your efforts!

To what are you referring? Disable what?

**Comment by calistoc** [comment]
Just a note about the admin page as it was driving me crazy for a while
After I updated from 1.21.0 to either 1.22.0 or 1.22.1 I thought there was a problem with the Admin page as clicking on “Settings->General Settings” was not doing anything, after a while I realized that I had to click a little below the “General Settings” and not over it like with the 1.21.0 version, the same happens with all other options in the Settings page, i.e “Advanced settings” “Yubikey settings”, etc..

Thanks for your work

**Reply by BlackDex** [reply]
@calistoc which browser and version are you using? And also, which OS?

**Reply by calistoc** [reply]
Chrome 91.0.4472.124 (Official Build) (64-bit) - Firefox 89.0.2 (64-bit) with Windows 10, here I see the problem.

I just tried with both Chrome & Firefox from Linux and it works fine

I'm running vaultwarden with docker on a Ubuntu Linux box
